src/screens/Target.tsx

```diff
@@ -29,7 +45,7 @@ const Target = ({ navigation }: NavigationProps): React.JSX.Element => {
             accessibilityLabel={accessibleText.target.input}
             style={input}
             placeholder="Digite sua meta em ML"
-            onChangeText={newText => setTarget(newText)}
+            onChangeText={newText => writeTargetToStorage({target: newText})}
```

Comment:
Faz sentido aqui a gente setar o `target` no `storeTarget` no `blur` do componente? Ou quando ele clicar em `começar`? Pq fico pensando se nao poder ser um problema chamar o `storeTarget` no evento do click

Reply:
é um ponto, se for no botão "Começar" vai ser um evento a menos. Como ele precisa ser clicado de qualquer forma, acho que dessa forma ficaria mais performático. Prosseguindo com a mudança.